lib/routes/scut/gzic/news.ts

```diff
@@ -0,0 +1,65 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/gzic/news',
+    categories: ['university'],
+    example: '/scut/gzic/news',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '广州国际校区 - 新闻聚焦',
+    maintainers: ['gdzhht'],
+    handler,
+    description: `:::warning
+    由于学校网站对非大陆 IP 的访问存在限制，可能需自行部署。
+    :::`,
+};
+
+async function handler() {
+    const baseUrl = 'https://www2.scut.edu.cn';
+    const url = `https://www2.scut.edu.cn/gzic/30279/list.htm`;
+
+    const { data: response } = await got(url);
+    const $ = load(response);
+
+    const list = $('.right-nr .row .col-lg-4')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            const a = item.find('.li-img a');
+            const pubDate = item.find('.li-img a span');
+            return {
+                title: item.find('.li-img a p').text(),
+                link: a.attr('href')?.includes('http') ? a.attr('href') : `${baseUrl}${a.attr('href')}`,
```

Comment:
`includes('http')` will be true even if http appears at the end of the string `a.attr('href')`. Please use `.startsWith()` instead.